src/form/ChestAPI/session/PlayerManager.php

```
@@ -0,0 +1,64 @@
+		$plugin_manager->registerEvent(PlayerLoginEvent::class, function(PlayerLoginEvent $event) : void{
+			$this->create($event->getPlayer());
+		}, EventPriority::MONITOR, $registrant);
+		$plugin_manager->registerEvent(PlayerQuitEvent::class, function(PlayerQuitEvent $event) : void{
+			$this->destroy($event->getPlayer());
+		}, EventPriority::MONITOR, $registrant);
+	}
+
+	private function create(Player $player) : void{
+		$this->sessions[$player->getId()] = new PlayerSession($player, new PlayerNetwork(
+			$player->getNetworkSession(),
+			$this->network_handler_registry->get($player->getPlayerInfo()->getExtraData()["DeviceOS"] ?? -1)
```

PHPStan (2.1.15)

Parameter #1 $os_id of method pocketmine\form\ChestAPI\session\network\handler\PlayerNetworkHandlerRegistry::get() expects int, mixed given.


(argument.type)